transport/src/main/java/io/netty/channel/local/LocalChannel.java

```diff
@@ -24,6 +24,7 @@
 import io.netty.channel.ChannelPromise;
 import io.netty.channel.DefaultChannelConfig;
 import io.netty.channel.EventLoop;
+import io.netty.channel.PreferHeapByteBufAllocator;
```

Comment:
nope it was moved the `io.netty.channel` (it was in the local package before) as we use it in OIO as well now.